frontend/src/containers/ConfigStep/DateRangePicker/index.tsx

```diff
@@ -1,18 +1,55 @@
-import { DateRangePickerGroup } from '@src/containers/ConfigStep/DateRangePicker/DateRangePickerGroup';
+import { DateRangePickerGroup, SortType } from '@src/containers/ConfigStep/DateRangePicker/DateRangePickerGroup';
+import { ISortedDateRangeType } from '@src/containers/ConfigStep/DateRangePicker/DateRangePickerGroup';
+import { selectDateRange, selectDateRangeSortStatus } from '@src/context/config/configSlice';
+import { SortDateRange } from '@src/containers/ConfigStep/DateRangePicker/SortDateRange';
 import SectionTitleWithTooltip from '@src/components/Common/SectionTitleWithTooltip';
+import { TitleContainer } from '@src/containers/ConfigStep/DateRangePicker/style';
 import { TIME_RANGE_TITLE, TIPS } from '@src/constants/resources';
+import { useAppSelector } from '@src/hooks/useAppDispatch';
+import { useMemo, useState } from 'react';
 
 export const DateRangePickerSection = () => {
+  const dateRangeGroup = useAppSelector(selectDateRange);
+  const dateRangeGroupSortStatus = useAppSelector(selectDateRangeSortStatus);
+  const [sortType, setSortType] = useState<SortType>(
+    dateRangeGroupSortStatus ? dateRangeGroupSortStatus : SortType.DEFAULT,
```

Comment:
SortType

Reply:
thanks for reminder, done, no `sortStatus` now